lib/CommandClient.php

```diff
@@ -114,33 +96,20 @@ public function stop(): Promise {
     }
 
     public function importServerSockets($addrCtxMap): Promise {
-        return \Amp\call(function() use ($addrCtxMap) {
+        return call(function() use ($addrCtxMap) {
             $reply = yield $this->send(["action" => "import-sockets", "addrCtxMap" => array_map(function ($context) { return $context["socket"]; }, $addrCtxMap)]);
 
-            // replace $this->socket temporarily by a dummy read() which will later be resolved to $this->socket->read() in order to avoid simultaneous read watchers here
```

Comment:
hmm, can that be removed? won't `$commandClient->imprtServerSockets(…); $commandClient->stop();` - just each immediately after the other cause problems? If there's an active read watcher from the next command?